migration-notice.md

```diff
@@ -0,0 +1,78 @@
+# Migration notice
+
+## CodePush SDK
+
+CodePush SDK migrated to a new service. We recommend updating CodePush SDK to the latest version. Versions are lower than **[4.0.0](https://github.com/microsoft/code-push/releases/tag/v4.0.0)** will not work in the near future.
+
+### Deprecated methods
+
+* getSessions
+* patchAccessKey
+* removeSession
+* getAccessKey
+
+These methods are not supported in the release's version **[4.0.0](https://github.com/microsoft/code-push/releases/tag/v4.0.0)** and above.
+
+### Comparison
+
+* Server URL
+
+  * versions **[4.0.0](https://github.com/microsoft/code-push/releases/tag/v4.0.0)** and above:
+
+    `https://api.appcenter.ms/v0.1`
+
+  * versions are lower than **[4.0.0](https://github.com/microsoft/code-push/releases/tag/v4.0.0)**:
+
+    `https://codepush-management.azurewebsites.net`
+
+    `https://codepush.appcenter.ms/v0.1/legacy`
+
+* Request's path
+
+    For example the path of `getDeployments` method:
+
+  * versions **[4.0.0](https://github.com/microsoft/code-push/releases/tag/v4.0.0)** and above:
+
+    `https://api.appcenter.ms/v0.1/apps/{owner_name}/{app_name}/deployments/`
+
+  * versions are lower than **[4.0.0](https://github.com/microsoft/code-push/releases/tag/v4.0.0)**:
+
+    `https://codepush-management.azurewebsites.net/apps/{app_name}/deployments/`
+
+* Error message
+
+  We change some messages for error handling. For example the output of `getDeployments` method:
```

Comment:
We should mention status code I suppose because it's usually important when handling network responses.

```suggestion
  Messages and status codes for error handling are different. For example the output of `getDeployments` method:
```